torch/lib/c10d/ProcessGroup.hpp

```diff
@@ -89,6 +89,10 @@ class ProcessGroup {
 
     virtual void abort();
 
+    // Returns a Future object that will be marked as completed once work is
```

Comment:
Isn't the future marked as completed on construction? I understand that the `completed()` API will return work's completed status. If this is correct let's make it explicit in the comment.